File: 1833-maximum-ice-cream-bars/1833-maximum-ice-cream-bars.c
int cmp(const void* a, const void* b){
    return(*(int*)a - *(int*)b);
}

int maxIceCream(int* costs, int costsSize, int coins){
    // sort arr
    qsort(costs, costsSize, sizeof(int), cmp);
    int f = 0, i = 0;
    while(i < costsSize && costs[i] <= coins){
        coins = coins - costs[i];
        i++; f++;
    }
    return f;
}

File: 1833-maximum-ice-cream-bars/1833-maximum-ice-cream-bars.py
class Solution:
    def maxIceCream(self, costs: List[int], coins: int) -> int:
        costs = sorted(costs)
        f, i = 0, 0
        l = len(costs)
        while i < l and costs[i] <= coins:
            coins -= costs[i]
            f += 1
            i += 1
        return f
        